vassal-app/src/main/resources/VASSAL/i18n/Editor.properties

```diff
@@ -1632,30 +1643,3 @@ Editor.Zoom.out_key=Zoom out hotkey
 Editor.Zoom.component_type=Zoom capability
 Editor.zoom.initial_zoom="An '*' indicates the initial zoom level."
 Editor.zoom.set_initial=Set Initial
-
-# Board
-Editor.Board.image=Board image:
-Editor.Board.reverse=Reversible:
-Editor.Board.width=Board width:
-Editor.Board.height=Board height:
-Editor.Board.component_type=Board
-
-# Grids
-Editor.Grid.y_offset=X offset:
-Editor.Grid.x_offset=Y offset:
-Editor.Grid.edges=Edges are legal locations?
-Editor.Grid.show_grid=Show Grid?
-Editor.Grid.center_dots=Draw Center Dots?
-Editor.Grid.edit_grid=Edit Grid
-Editor.Grid.snap=Snap to defined point?
-
-# Grid Editor
-Editor.GridEditor.arrow_keys=Arrow Keys - Move Grid
-Editor.GridEditor.cancel_set=Cancel Set
-Editor.GridEditor.click_on_3=Click on 3 adjacent points around the edge of any map grid cell
-Editor.GridEditor.control_arrow_keys=Control-Arrow Keys - Resize Grid
-Editor.GridEditor.grid_shape_error=Grid Shape Error
-Editor.GridEditor.numbering=Numbering
-Editor.GridEditor.set_grid_shape=Set Grid Shape
-Editor.GridEditor.shift_key=Shift Key - Increase speed of other keys
-Editor.GridEditor.does_not_look=Does not look like a %1$s!
```

Comment:
Where are all these going?

Reply:
These are all duplicates that got merged in with the chess clock PR. The real ones are up in their correct alphabetically sorted place. The Chess Clock PR was created when Editor.properties had stuff all over the place.